app/controllers/concerns/metadata.rb

```diff
@@ -12,7 +12,8 @@ def set_headers
       response.headers['Content-Type'] = 'application/vnd.api+json'
     end
 
-    def metadata(opts)
+    def metadata(opts = {})
+      opts[:total] ||= policy_scope(resource).count
```

Comment:
This is just a convenience, before this, I had to call this method on every controller

Reply:
I think we should include `limit` and `offset` in the `meta` option as well so the user sees what the defaults are if nothing is provided, and they see what the server received if they do provide limit and offset.